packages/@o3r/schematics/src/utility/wrapper.ts

```diff
@@ -0,0 +1,36 @@
+import type { Rule } from '@angular-devkit/schematics';
+import { performance } from 'node:perf_hooks';
+import { getEnvironmentInfo } from './environment';
+
+export const createSchematics = <S>(
```

Comment:
missing comment.
Maybe would should also give it a name more explicite to indicate it's usage (something like `createSchematicWithMetrics`)

Reply:
renamed